### DISCUSSION TITLE
Getting Started: Which Technology Stack Should We PIck & Why?

### DISCUSSION THREAD

**Discussion by vinodsharma10x** [careerJumpstart/TestimonialFree#4] [Frequently Asked Q&A] [answered]
Let's pick the technology stack together.

# This is what I have in mind. #

1/ HTML, CSS, JavaScript
2/ React
3/ NextJs - a framework for building server-rendered React applications
4/ NestJs
5/ Maybe using Python or C# for backend / API code
6/ I don't know which database we should use. One option is to use the Free version of MS SQL.

Other options
MongoDB - a NoSQL database
Tailwind CSS - a utility-first CSS framework

# Code editor #

I suggest that we use VS Code to edit the code

Please share your thoughts and decide as a group.

**Comment by roberto-morado** [comment]
I like the stack 🎉

I'm not sure about the site's dynamics or UX/UI but I found this social network tailwindcss template:

https://tailwindcomponents.com/component/social-feed

I believe we can brake it down into components and use it as leverage.

**Comment by roberto-morado** [comment]
Is there a plan to move forward? managing tasks, docs, choosing stack, voting, assigning roles, etc?

Otherwise I'd like to, at least, setup the project, unless you are already working on it.

The proposed stack looks good enough, it could very well help us reach some milestones

**Comment by vinodsharma10x** [comment]
I had a crazy week and next week will be the same. 

I want to arrange a zoom call with everyone interested and decide on tech.

Robert, you have been very active. Would you like to take the lead in setting up the project?

I can jump on a zoom call today or tomorrow to meet if you like or else you are clear on the steps then go ahead.

**Reply by roberto-morado** [reply]
Yes, I'd be more than glad.

I'm also about to finish a development plan for the project. I spend the weekend working on it; it is certainly opinionated based on the information you have provided so far, but I believe is a starting point; you mind if I create a pull request later today?

I'm not sure if I require any permissions to push to the repository, I'll try anyway. 🎉

**Reply by roberto-morado** [reply]
I'd be busy till later tonight but if it works bets for you we can meet tomorrow? how's your schedule looking?

**Comment by roberto-morado** [comment]
It'd be useful if you could give me permissions to push to the repository. 👍

**Reply by vinodsharma10x** [reply]
Done. I added you as a maintain. You should have all access now.

**Comment by vinodsharma10x** [comment, marked as answer]
Sorry I was out and just came back home. Giving you access.

**Reply by roberto-morado** [reply]
Thank you; I'll open that pr so that you can check it out.

**Comment by roberto-morado** [comment]
### #7 is now open

Hope it helps

**Reply by vinodsharma10x** [reply]
Thank you. It looks good. Please check this page as well where I try to write about the project https://github.com/careerJumpstart/TestimonialFree/wiki

**Comment by ClaraLopezUribe** [comment]
Hello!

May I selfishly put in my vote for a few of these technologies?  I would love to apply what I am learning in my self-paced .Net boot camp directly to this project:

* Along with CSS,  I am also using Bootstrap (it has very good documentation)
* C#
* ASP.NET MVC
* PostgreSQL and pgAdmin

Thanks!
